db/crud.go

```diff
@@ -531,7 +533,11 @@ func (db *Database) Put(docid string, body Body) (newRevID string, err error) {
 		// Make up a new _rev, and add it to the history:
 		newRev := createRevID(generation, matchRev, body)
 		body["_rev"] = newRev
-		doc.History.addRevision(RevInfo{ID: newRev, Parent: matchRev, Deleted: deleted})
+		if err := doc.History.addRevision(RevInfo{ID: newRev, Parent: matchRev, Deleted: deleted}); err != nil {
+			base.LogTo("Import+", "Import failed to add revision ID: %s, error: %v", newRev, err)
```

Comment:
As above - this isn't only during import - should just log as CRUD and remove 'Import' from the log message.